bsp_diff/caas/device/intel/civ/host/vm-manager/0046-Multi-Android-VM-support.patch

```diff
@@ -0,0 +1,407 @@
+From 3cac426b2b7995bcf5959ae2373520b0b601a177 Mon Sep 17 00:00:00 2001
+From: "Suresh, Prashanth" <prashanth.suresh@intel.com>
+Date: Thu, 15 Jun 2023 12:13:48 +0530
+Subject: [PATCH] Multi Android VM support
+
+Tracked-On: OAM-110639
+
+Signed-off-by: Suresh, Prashanth <prashanth.suresh@intel.com>
+---
+ scripts/create_vms.sh                     |  53 ++++++++++++
+ scripts/setup_civ_ini.sh                  |  47 ++++++++--
+ scripts/setup_host.sh                     |  51 ++++++++++-
+ scripts/setup_host_multiple_android_vm.sh | 101 ++++++++++++++++++++++
+ scripts/start_flash_usb.sh                |  22 ++++-
+ 5 files changed, 260 insertions(+), 14 deletions(-)
+ create mode 100644 scripts/create_vms.sh
+ create mode 100644 scripts/setup_host_multiple_android_vm.sh
+
+diff --git a/scripts/create_vms.sh b/scripts/create_vms.sh
+new file mode 100644
+index 0000000..fbb5197
+--- /dev/null
++++ b/scripts/create_vms.sh
+@@ -0,0 +1,53 @@
++#!/bin/bash
++
++# Copyright (c) 2023 Intel Corporation.
++# All rights reserved.
++#
++# SPDX-License-Identifier: Apache-2.0
++
++set -eE
++
++VSOCK_ID=3
++ADB_PORT=5555
++VM_NAME=Android-CIV
++CIV_WORK_DIR=$(pwd)
++
++function create_vm_dirs() {
++    echo "Dir: $CIV_WORK_DIR/$VM_NAME"
++    mkdir $CIV_WORK_DIR/$VM_NAME
++    $CIV_WORK_DIR/scripts/setup_civ_ini.sh -v $1 -p $2 -n $3
++}
++
++function copy_files_for_vm() {
++    echo "Copying file: $CIV_WORK_DIR/$VM_NAME"
++    mkdir $CIV_WORK_DIR/$VM_NAME/scripts
++    cp -r $CIV_WORK_DIR/scripts/aaf $CIV_WORK_DIR/$VM_NAME/scripts/
++    cp OVMF.fd $CIV_WORK_DIR/$VM_NAME/
++    cp $CIV_WORK_DIR/scripts/rpmb_dev $CIV_WORK_DIR/$VM_NAME/scripts/
```

Comment:
For the copy operations, check that files exist otherwise display error message and exit